assets/index.html

```diff
@@ -0,0 +1,246 @@
+        <div class="hero__cta">
+          <h1 class="hero__main-header">Beautiful Free Nova template</h1>
+          <p class="hero__content">
+            A top notch premium quality template for your next web project.
+          </p>
+          <button class="btn btn--hero">Download FREE!</button>
+        </div>
+        <div class="hero__image">
+          <img
+            src="../images/screen.png"
+            alt=""
+          />
+        </div>
+      </div>
+    </header>
+
+    <section class="features">
+      <div class="wrapper">
+        <h2 class="features__title">features</h2>
+      </div>
+
+      <div class="wrapper">
+        <div class="card">
+          <div class="card__circle">
+          <img src="../images/responsive.svg" class="features__icon"></img>
+          </div>
+          <h4 class="card__title">Fully Responsive</h4>
+          <p class="card__content">Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis, quas.</p>
+        </div>
+        <div class="card">
+          <div class="card__circle">
+          <img src="../images/customizable.svg" class="features__icon"></img>
+        </div>
+          <h4 class="card__title">Customizable</h4>
+          <p class="card__content">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Optio, officia?</p>
+        </div>
+        <div class="card">
+          <div class="card__circle">
+            <img src="../images/design.svg" class="features__icon"></img>
+          </div>
+          <h4 class="card__title">Slick and beautiful design</h4>
+          <p class="card__content">Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus, architecto!</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="logos">
+      <div class="wrapper">
+        <img src="../images/logos.png" alt="" class="logos__image">
+      </div>
+    </section>
+
+    <section class="about">
+      <div class="image-container">
+        <img src="../images/imac.png" alt="imac">
+      </div>
+      <div class="content-container">
+        <h2 class="about__title">simple and beautiful</h2>
+        <p class="about__content">
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quas, ullam
+          dolorem suscipit nihil, voluptatum perspiciatis facere, provident
+          consequatur alias molestias quidem facilis fugiat nisi quia
+          repudiandae reiciendis commodi libero rem? Rem incidunt architecto
+          doloremque soluta mollitia odit hic quia, labore velit assumenda
+          molestias voluptas est eius esse facere, nostrum fugit?
+        </p>
+      </div>
+    </section>
+
+    <section class="pricing">
+      <div class="wrapper">
+        <h2 class="pricing__title">Choose your pricing plan</h2>
+        <p class="pricing__text">Pick any of our super affordable pricing plans</p>
+      </div>
+
+      <div class="wrapper card-wrapper">
+        <div class="subscription">
```

Comment:
Tutaj bym zrobić też `ul > li` jako lista pakietów lub dla `.subscription` dał bym `<article>` - bo to jedna spójna całość.